templates/gatsby-strapi/files/gatsby/gatsby-config.js

```diff
@@ -1,7 +1,7 @@
 const config = require("platformsh-config").config();
 
 var backend_route = "";
-if ( config.isValidPlatform() ) {
+if ( config.isValidPlatform() && !config.inBuild()) {
```

Comment:
Extra space before `config`.
(I know there were spaces at both sides of the expression before but there's no space for the same code in `templates/gatsby-strapi/files/strapi/config/database.js` L15 so I think it's better to be consistent.)